uwblueprint/jump-math
# feat: fetch data for test sessions page

## Notion ticket link

[Fetch data for Test Sessions page](https://www.notion.so/uwblueprintexecs/Fetch-data-for-Test-Sessions-page-29a10c0026654c0ab2ba0a19db62e86f)

Strongly suggest reviewing by commit!

- Fetch data from backend
- Fetch className from backend
- Fix linter script on Mac OS

<img width="1398" alt="image" src="https://github.com/uwblueprint/jump-math/assets/9922514/16749278-3747-4830-b311-60e07c61bfe6">




## Implementation description
* As discussed on Slack, decided to add a reference to the classroom into the session object. Alternatively would require multiple fetches.



## Steps to test
1. View the teacher session page and verify that it loads and you see live data (probably nothing :P).



## What should reviewers focus on?
* 


## Checklist
